.travis.yml

```diff
@@ -1,16 +1,18 @@
 language: python
 
-python:
-    - "2.7"
+python: "2.7"
 
 sudo: false
 
-install: 
-    - "pip install -q -r requirements.txt --use-mirrors"
+cache:
+    directories:
+        - $HOME/.pip-cache/
+        - _build
+
+install: pip install --download-cache $HOME/.pip-cache --user sphinx==1.1.3
```

Comment:
there was no need for a special requirements file. And as pip now has a cache configured, it'll skip the download on all builds.